src/main/java/bio/terra/cli/command/Main.java

```diff
@@ -16,6 +16,7 @@
     name = "terra",
     subcommands = {
       Status.class,
```

Comment:
nit: alphabetize this list

Reply:
Agreed. Planning to tackle this, along with other style improvements, in a separate ticket ([PF-574](https://broadworkbench.atlassian.net/browse/PF-574)).